prometheus/auto.d/ReplicaDB.yml.example

```diff
@@ -1,3 +1,4 @@
 - targets: [ "10.123.202.13:9100", "10.123.202.13:9187" ]
   labels:
     job: "Staging"
```

Comment:
Calling this "staging" in the replica file is a bit confusing. Change job name to "Replica"?

Also, need to remove the node_exporter target from here.

Reply:
Fixed in https://github.com/CrunchyData/pgmonitor/pull/89/commits/2cf98aa2d8ae00d6c8c10707d86194af6cc0adf7